Assets/NuGet/Editor/NugetHelper.cs

```diff
@@ -1383,7 +1498,8 @@ public static Texture2D DownloadImage(string url)
             stopwatch.Start();
 
             bool fromCache = false;
-            if (ExistsInDiskCache(url)) {
+            if (ExistsInDiskCache(url))
```

Comment:
Please don't apply sweeping formatting changes and adhere to the formatting in place.

Reply:
Sorry about that. I didn't follow the formatting and I fixed that already.